gb-bus/src/io_reg_area.rs

```diff
@@ -0,0 +1,38 @@
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum IORegArea {
+    Controller,
+    Communication,
+    DivTimer,
+    Sound,
+    WaveformRam,
+    Lcd,
+    VRamBank,
+    BootRom,
+    VramDma,
+    BgObjPalettes,
```

Comment:
What do you think of this name ?

Reply:
I think you can merge the `BgObjPalettes` and the `VramDma` into the `Lcd` section, since they are on a continuous area. See #74.

Reply:
My bad I've just checked #68, I was confused between the content of `Lcd` and `VramDma`/`BgObjPalettes`. The words Palettes and DMA both designate registers inside `Lcd` and separates registers for the CGB.

Reply:
so, the name are ok for you or you want to merge them ?

Reply:
I think they're ok, I was hesitant to ask for a `CGB` prefix for cgb specific registers areas, but now that I've realized the differences between the lcd registers and theses area I don't think its necessary.

Reply:
So I mark it as resolved